common/web/lm-worker/build.sh

```diff
@@ -52,48 +52,36 @@ wrap-worker-code ( ) {
   echo "// --START:LMLayerWorkerCode--"
   printf "function %s () {\n" "${name}"
 
-  # TODO: I don't know if this is still true, or if we can add the polyfills to
-  # the project directly now?
```

Comment:
Hmm.  We could test this, I suppose.  It's possible the declaration emitter got enhanced in the newer TS version we're using now.

It definitely fell over quickly when I tried directly linking against it with `allowJS=true` in the config before we were using incremental builds.